src/Catalog/ProductSummary.php

```diff
@@ -205,4 +205,18 @@ public function getGeolocation(): ?ProductLocation
     {
         return $this->geolocation;
     }
+
+    public function getSingleDeclinationId(): ?DeclinationId
```

Comment:
Nouveau getter qui contient la logique de check de `$this->getDeclinationCount()`.

Du coup dans les apps front, il "suffit" de checker si ce getter renvoi `null` ou pas pour savoir si on peut ajouter au panier/favoris.

@ngdo-pro ca te semble bien ?

Reply:
On en discute lundi ? Perso je suis pas convaincu mais je veux bien me laisser convaincre :p